rails/ordinals/fr-CA.rb

```diff
@@ -0,0 +1,19 @@
+{
+  :"fr-CA": {
```

Comment:
👋 I'm getting a syntax error on this, not sure why CI didn't catch it:

```ruby
>> {:"fr-CA": nil}
`eval': (irb):1: syntax error, unexpected ':', expecting => (SyntaxError)

Reply:
Oh darn. Tests cover only the `fr.rb` file. I’ll add a failing test for all locales.